sdcm/cluster.py

```diff
@@ -2889,21 +2894,21 @@ def wait_for_machine_image_configured(self):
         wait.wait_for(self.is_machine_image_configured, step=10, timeout=300)
 
     def get_sysctl_output(self) -> Dict[str, str]:
```

Comment:
Maybe rename it to get_sysctl_properties

Reply:
done